api/src/main/resources/messages.properties

```diff
@@ -1,3 +1,4 @@
 ${project.parent.artifactId}.title=Location based access control module
 
 locationbasedaccess.patientLocation=Patient Location
```

Comment:
Are we still using locationbasedaccess.patientLocation for anything?

Reply:
For showing the label on the patient registration dashboard.